tools/release/changelog-renderer.js

```diff
@@ -2,34 +2,15 @@
 
 // @ts-check
 const {
-  default: defaultChangelogRenderer,
+  default: DefaultChangelogRenderer,
 } = require('nx/release/changelog-renderer');
 
-const changelogRenderer = async ({
-  projectGraph,
-  commits,
-  releaseVersion,
-  project,
-  entryWhenNoChanges,
-  changelogRenderOptions,
-  repoSlug,
-  conventionalCommitsConfig,
-  changes,
-}) => {
-  const defaultChangelog = await defaultChangelogRenderer({
-    projectGraph,
-    commits,
-    releaseVersion,
-    project,
-    entryWhenNoChanges,
-    changelogRenderOptions,
-    repoSlug,
-    conventionalCommitsConfig,
-    changes,
-  });
-
-  // Append our custom messaging to the generated changelog entry
-  return `${defaultChangelog}\n\nYou can read about our [versioning strategy](https://main--typescript-eslint.netlify.app/users/versioning) and [releases](https://main--typescript-eslint.netlify.app/users/releases) on our website.`;
+module.exports = class CustomChangelogRenderer extends (
```

Comment:
The changelog renderer API is now class based where individual elements of the render can be easily patched by overwriting base methods. In our case we are just bolting text onto the full default render, so we only need to amend the main render() method.